actionpack/lib/action_controller/metal/mime_responds.rb

```diff
@@ -296,8 +292,8 @@ def initialize(variant = nil)
         end
 
         def any(*args, &block)
-          if block_given?
-            if args.any? && args.none? { |a| a == @variant }
+          if block
+            if args.any? && args.none?(@variant)
```

Comment:
This replacement seem dangerous to me, AFAIK `none?` will use `===` for comparison, that may lead to some things matching when they shouldn't. 

It may be safe in the current context, that need to be audited, but rubocop has no business doing this replacement automatically.

Reply:
So please disable that cop so other people don't get tripped by it.